app/Link.php

```diff
@@ -36,8 +36,12 @@ public function member()
 
     public static function rules()
     {
+        // Spaces in front of commas removed because they cause validation to become wonky
         return [
-            'contact_options' => 'required',
+            'show_home_phone' => 'required_without_all:show_uk_phone,show_other_email,show_hermes_email',
+            'show_uk_phone' => 'required_without_all:show_home_phone,show_hermes_email,show_other_email',
+            'show_hermes_email' => 'required_without_all:show_home_phone,show_other_email,show_uk_phone',
+            'show_other_email' => 'required_without_all:show_uk_phone,show_hermes_email,show_home_phone',
         ];
     }
-}
+}
```

Comment:
Great in general to be consistent and to end lines with a newline (: See http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more discussion haha on that